tests/FVTTests/fileParameterView/testFileParameterView.ts

```diff
@@ -70,17 +66,25 @@ describe('JES explorer spool file in url query (explorer-jes/#/viewer)', functio
         await loadUrlWithSearchFilters(driver, filters);
         const jobObjs :ParsedJobText[] = await waitForAndExtractParsedJobs(driver);
         expect(jobObjs && jobObjs.length > 0).to.be.true;
+
         testFilters = {
             jobName: jobObjs[0].prefix,
             jobId: jobObjs[0].jobId,
-            fileId: 3,
-        };
+            fileId: 108,
+        }
+
+        // load driver with specified filter
         await loadUrlWithViewerFilters(driver, testFilters);
+ 
+    });
 
-        // wait for content to load
-        const viewer = await driver.findElement(By.css('#embeddedEditor > div > div > .textviewContent'));
-        const text = await viewer.getText();
-        expect(text).to.have.lengthOf.greaterThan(1);
+    it('Should handle rendering file contents in Orion editor', async () => {
+        // wait for content to load and check if the file is open correctly with specified strings
+        let viewer = await driver.findElement(By.css('#embeddedEditor > div > div > .textviewContent'));
+        let text = await viewer.getText();
```

Comment:
Rather than calling text.trim() twice in the two lines following this, can we trim on this line to remove duplication below.

Reply:
Completely agree.